### PR TITLE
sui: add page

### PR DESCRIPTION
**Pull request by Tzal3x**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):** 1.36.2

### CODE REVIEW

File: pages/common/sui-move.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+- Build the Move project in the current directory:
+
+`sui move build`
+
+- Build the Move project from the given path:
+
+`sui move build --path {{path}}`
+
+- Migrate to Move 2024 for the package at provided path:
```
Comment (spageektti) [inline pages/common/sui-move.md:30]:
```suggestion
- Migrate to Move 2024 for the package at the provided path:
```

File: pages/common/sui-client.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# sui client
+
+> Publish smart contracts, get object information, execute transactions and more.
```
Comment (spageektti) [inline pages/common/sui-client.md:3]:
```suggestion
> Publish smart contracts, get object information, execute transactions, and more.
```

File: pages/common/sui-client.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+> More information: <https://docs.sui.io/references/cli/client>.
+
+- Create a new address with ED25519 scheme:
```
Comment (spageektti) [inline pages/common/sui-client.md:6]:
```suggestion
- Create a new address with the ED25519 scheme:
```

File: pages/common/sui-client.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+- Interact with the Sui faucet:
+
+`sui client faucet {{subcommand}}`
+
+- List the gas coins for the given address (accepts also an alias):
+
+`sui client gas {{address}}`
+
+- Create, sign and execute programmable transaction blocks:
```
Comment (spageektti) [inline pages/common/sui-client.md:34]:
```suggestion
- Create, sign, and execute programmable transaction blocks:
```

**Comment by spageektti** [comment]
Hi, thanks for your contribution. The descriptions of examples should be written in the imperative mood (see [style guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#imperative-mood)).

File: pages/common/sui.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+- Execute a sui subcommand:
+
+`sui {{subcommand}}`
+
+- Build tools for a smart contract:
+
+`sui move {{subcommand}}`
+
+- Client to interact with the Sui network:
```
Comment (spageektti) [inline pages/common/sui.md:14]: The descriptions of examples should be written in the imperative mood (see [style guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#imperative-mood)).

Comment (Tzal3x) [reply pages/common/sui.md]: Missed this one, thanks a lot! I updated it. ✅

**Review by nelsonfigueroa — APPROVED** [review]

**Review by spageektti — APPROVED** [review]

**Pull request merged**